### DISCUSSION TITLE
Binder launch button missing

### DISCUSSION THREAD

**Discussion by thiippal** [executablebooks/meta#230] [Q&A]
Hey!

I've been banging my head against the wall with this problem for several days already: I simply cannot get the Binder launch button to appear in my documentation using Sphinx book theme: https://applied-language-technology.readthedocs.io/

All other buttons for interactivity work just fine except for the Binder launch button, which does not show up.

Here's the link to my `conf.py`: https://github.com/Applied-Language-Technology/website/blob/master/source/conf.py

If someone has encountered a similar problem and solved it, I would greatly appreciate any help!

**Comment by SDSTony** [comment]
It seems you have inserted the default binder url in your `conf.py` file which is mentioned on the [Jupyter Book Docs](https://jupyterbook.org/interactive/launchbuttons.html?highlight=interactivity#binder-buttons-for-your-pages). 

Try switching the default url to the url link that the [binder](https://mybinder.org/) creates for you when you build your repository as shown in the picture below. 
![image](https://user-images.githubusercontent.com/46511987/106673765-79d00180-65f5-11eb-9931-7620344116dc.png)

1. Enter the repo with `.ipynb` files that you want to host on binder into A and press the launch button. It will create a binder space in the link shown in B. 
2. Then add the link from B to your `conf.py` file. 

Hope it helps.

**Reply by thiippal** [reply]
Hi @SDSTony!

Thanks for your help – I somehow assumed that the Sphinx book theme would automatically create the URL for Binder from the other variables in `conf.py`.

```
html_theme_options = {
	"use_download_button": True,
	"use_repository_button": True,
	"use_edit_page_button": True,
	"repository_url": "https://github.com/Applied-Language-Technology/website/",
	"repository_branch": "master",
	# "path_to_docs": "source/",
	"launch_buttons": {"binderhub_url": "https://mybinder.org/v2/gh/Applied-Language-Technology/notebooks/main",
					    "thebe": False,
					    "notebook_interface": "jupyterlab",
					    "collapse_navigation": False}
	}
```

I added the Binder URL under the `launch_buttons` variable, but still can't get the launch button to show up.
